layouts/partials/head.html

```diff
@@ -0,0 +1,95 @@
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+{{ hugo.Generator }}
+{{- $outputFormat := partial "outputformat.html" . -}}
+
+{{ range .AlternativeOutputFormats -}}
+<link rel="{{ .Rel }}" type="{{ .MediaType.Type }}" href="{{ .Permalink | safeURL }}">
+{{ end -}}
+
+{{ if and hugo.IsProduction (ne $outputFormat "print") -}}
+<META NAME="ROBOTS" CONTENT="INDEX, FOLLOW">
+{{ else -}}
+<META NAME="ROBOTS" CONTENT="NOINDEX, NOFOLLOW">
+{{ end -}}
+
+{{ partialCached "favicons.html" . }}
+<title>{{ if .IsHome }}{{ .Site.Title }}{{ else }}{{ with .Title }}{{ . }} | {{ end }}{{ .Site.Title }}{{ end }}</title>
+{{- template "_internal/opengraph.html" . -}}
+{{- template "_internal/google_news.html" . -}}
+{{- template "_internal/schema.html" . -}}
+{{- template "_internal/twitter_cards.html" . -}}
+{{ if hugo.IsProduction }}
+{{ template "_internal/google_analytics_async.html" . }}
+{{ end }}
+{{ partialCached "head-css.html" . "asdf" }}
+<script
+  src="https://code.jquery.com/jquery-3.5.1.min.js"
+  integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0="
+  crossorigin="anonymous"></script>
+{{ if .Site.Params.offlineSearch }}
+<script
+  src="https://unpkg.com/lunr@2.3.8/lunr.min.js"
+  integrity="sha384-vRQ9bDyE0Wnu+lMfm57BlYLO0/XauFuKpVsZPs7KEDwYKktWi5+Kz3MP8++DFlRY"
+  crossorigin="anonymous"></script>
+{{end}}
+{{ if .Site.Params.prism_syntax_highlighting }}
+<!-- stylesheet for Prism -->
+<link rel="stylesheet" href="{{ "/css/prism.css" | relURL }}"/>
+{{ end }}
+{{ partial "hooks/head-end.html" . }}
+{{/* Everything above here is copied from themes/docsy/layouts/partials/head.html to add the below section */}}
+<script type="text/javascript" charset="utf8" src="https://cdn.datatables.net/1.10.25/js/jquery.dataTables.js" integrity="sha384-9jwHdz3xcZ/aQmgwzEl+IfbIibihtBOx76/CoExUWGC11Nb4blmMdE8y2edqF9EF" crossorigin="anonymous"></script>
+<script>
+function format ( d ) {
+    return '<table cellpadding="5" cellspacing="0" border="0" style="padding-left:50px;">'+
+        '<tr>'+
+            '<td>RELEASE IMAGE:</td>'+
+            '<td>'+d.releaseImage+'</td>'+
+        '</tr>'+
+        '<tr>'+
+            '<td>LABELS:</td>'+
+            '<td>'+d.labels+'</td>'+
+        '</tr>'+
+    '</table>';
+}
+
+  $(document).ready( function () {
```

Comment:
All of this seems awfully complicated and hard to maintain. Can't we use [`getJSON`](https://gohugo.io/templates/data-templates/#get-remote-data) and do it all in a go template?